assets/components/productPage/productPagePlanForm/productPagePlanForm.jsx

```diff
@@ -16,7 +17,9 @@ import './productPagePlanForm.scss';
 export type Plan = {|
   title: string,
   copy: string,
-  offer: string | null,
+  offer: Option<string>,
+  price: Option<string>,
```

Comment:
might be good to name `price` and `saving` something more data agnostic like `footerHeader` and `footerCopy`?